suse-open-source-policy.html

```diff
@@ -179,13 +179,23 @@ <h3 id="managing-source-code">Managing Source Code</h3>
 
     <h3 id="factory-first">Factory First</h3>
 
-    <p>When packaging new Open Source software or adding changes to existing packages follow the “Factory First” model. “Factory” is the common code stream all our distributions are based on. It is the immediate source for openSUSE Tumbleweed and it eventually ends up in openSUSE Leap and the SUSE Linux Enterprise distributions. This is a continuation of the “Upstream First” principle on the distribution level. It reduces maintenance effort and leverages the community.</p>
+    <p>When packaging new Open Source software or adding changes to existing packages follow the "Factory First" model. <a href="https://susedefines.suse.com/definition/opensuse-factory/">"Factory"</a> is the common code stream all our distributions are based on. It is the immediate source for openSUSE Tumbleweed and it eventually ends up in openSUSE Leap and the SUSE Linux Enterprise distributions. This is a continuation of the "Upstream First" principle on the distribution level. It reduces maintenance effort and leverages the community.</p>
 
-    <p>“Factory First” extends to other SUSE products. Packages for all products should generally come from the common code stream our distributions are based on. So for packages you need in any products contribute them to factory first.</p>
+    <p>"Factory First" extends to other SUSE products. Packages for all products should generally come from the common code stream our distributions are based on. So for packages you need in any products contribute them to factory first.</p>
+
+    <h3 id="first-summary">"First" summary</h3>
+
+    <p>"Upstream first" applies to all individual upstream projects, such as the Linux kernel and nearly any package on SLE.</p>
+
+    <p>"Upstream first" also applies when there is an estalished upstream distribution, such as Tumbleweed (for SUSE Linux Enterprise) or Uyuni (for SUSE Manager)</p>
+
+    <p>"Factory first" applies to the SUSE Linux Enterprise family. It is recommended for things tightly connected to SUSE Linux Enterprise. It should also be considered (not necessarily adopted) for every SUSE product.</p>
+
+    <p>Creating an "intermediate upstream" distribution in between an existing upstream project (like Ceph, Kubernetes or Cloud Foundry) and a SUSE product is not recommended for nor against. It should be assessed on a case by ase base without bias, taking benefits and costs into account.</p>
```

Comment:
There is one typo in the last sentence (case by case base). For the first sentence the last part isn't clear to me, what it means ("is not recommended for nor against"

Reply:
So are we're abandoning our commitment to openSUSE? 

This clause seems to dramatically water down our commitments compared to what we've told the communtiy to date and what we've actually done to date.

It's worth considering that every currently sold SUSE product currently relies on Factory First.

It's also worth noting that openSUSE-based "intermediate upstreams" have been made for practically every product we've ever sold successfully. eg Kubic for CaaSP, Uyuni (wholly dependant on openSUSE Leap and openSUSE infra) for Manager, Tumbleweed for SES, Tumbleweed for HA, Tumbleweed for GEO clustering, MicroOS for SLE Micro..)

I think that any undermining of our previously stated policies with regards to community contribution must be done in consultation with those communities. Has the openSUSE Board been consulted? @GeraldPfeifer ?

Reply:
I would be very concerned by this statement as a Board member, since it gives the impression that investment in openSUSE is discouraged. In general, the creation of intermediate products between SUSE and upstream in the openSUSE Project has created a ton of value by allowing a community to build around the solutions and services SUSE is building.

Reply:
> I would be very concerned by this statement as a Board member, since it gives the impression that investment in openSUSE is discouraged. In general, the creation of intermediate products between SUSE and upstream in the openSUSE Project has created a ton of value by allowing a community to build around the solutions and services SUSE is building.

That's definitely not the case: It's rather the opposite, the goal of SUSE is to help openSUSE grow its user base, and to help it address more use cases well. SUSE will support that goal with both resources and hardware investment as needed.

Reply:
Good to know, thanks.

Reply:
To make it more exact: It's a goal that I've set for my organization (SUSE Linux Systems Group).

Reply:
> > I would be very concerned by this statement as a Board member, since it gives the impression that investment in openSUSE is discouraged. In general, the creation of intermediate products between SUSE and upstream in the openSUSE Project has created a ton of value by allowing a community to build around the solutions and services SUSE is building.
> 
> That's definitely not the case: It's rather the opposite, the goal of SUSE is to help openSUSE grow its user base, and to help it address more use cases well. SUSE will support that goal with both resources and hardware investment as needed.

I have heard this from you and several others, however in the world of open source actions speak far louder then words and this is an action that could be seen in a negative light by the community at a time when you are saying that you want to grow openSUSE but yet the community is not seeing any fruit from this promise. Until the community can see evedance that we in (SUSE Linux Systems Group) are trying to Grow openSUSE many will struggle to believe it. Actions such as this don't help I strongly suggest that we make changes to the policy to better clarify and give understanding to what we are trying to achieve because at the moment we are making it very easy for the community to take something that shouldn't affect them at all quite negatively